### PR TITLE
Cleanup readme examples a bit, fix a couple small errors, add license info

### PR DESCRIPTION
**Pull request by konklone**
This fixes a couple small errors in the instructions, rewords examples a bit for clarity, and adds the standard 18F license boilerplate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,15 @@ You'll need to use some sort of strategy to export the environment variables in
 git clone git@github.com:18F/tock-management-reports.git
 cd tock-management-reports
 cp .env.example .env
-pyvenv env
+pyenv env
```
Comment (vdavez) [inline README.md:27]: This is actually [pyvenv](https://docs.python.org/3/library/venv.html) which comes native as of python 3.3 and is apparently deprecated as of 3.6. Now it should read `python -m venv env`.

**Review by vdavez — APPROVED** [review]
Going to merge and make the one pyvenv change. Thanks!

**Comment by vdavez** [comment]
Thanks @konklone!

File: README.md (hunk 1 above)
Comment (konklone) [reply README.md:27]: omg

**Pull request merged**